lib/feedjira/logger.rb

```diff
@@ -0,0 +1,79 @@
+# Feedjira.logger
+module Feedjira
+  class << self
+    attr_writer :logger
+
+    # Provides a global logger instance for Feedjira
+    # If no logger is set by the user, a logger using DEFAULTS is provided
+    # @see Feedjira::Config::DEFAULTS
+    # @see Feedjira::Logger
+    #
+    # @return [Logger]
+    def logger
```

Comment:
For ruby's logger you have the choice over taking a block or using a method.
Take a block is the more efficient way because it's only called, if the log message needs to be written. 

Let's say you've set severity to "Info" and the code produces a log message for debug mode. When calling a method, the Message will be evaluated even if you don't need them. Wen using a block, the Message will only be evaluated if the severity is set to Debug.

See also [How to log a message](http://ruby-doc.org/stdlib-2.1.0/libdoc/logger/rdoc/Logger.html#class-Logger-label-How+to+log+a+message)